src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrentSettings.cs

```diff
@@ -63,6 +65,15 @@ public QBittorrentSettings()
         [FieldDefinition(10, Label = "Initial State", Type = FieldType.Select, SelectOptions = typeof(QBittorrentState), HelpText = "Initial state for torrents added to qBittorrent")]
         public int InitialState { get; set; }
 
+        [FieldDefinition(11, Label = "Remove stalled downloads", Type = FieldType.Checkbox, Advanced = true, HelpText = "Remove stalled downloads from the client. BEWARE: This could get you banned from trackers!")]
+        public bool RemoveStalledDownloads { get; set; }
```

Comment:
Do we need this third setting, or can we use 0 or blank on the below two settings as a "Don't remove" option?

Reply:
My only concern about it, specially if we do decide to use a more coarse unit for the time, is that we take away from the user the scenario where they want to remove it immediately.